py3status/modules/ns_checker.py

```diff
@@ -33,16 +40,21 @@ class Py3status:
     """
     """
     # available configuration parameters
+    cache_timeout = 300
     domain = ''
+    format = '{ns_total} NS {ns_status}'
```

Comment:
You haven't added this to the docstring but to be honest it might be good to wait till #499 is merged due to docstring conflicts.